README.md

```diff
@@ -27,9 +32,32 @@ Directory `src` contains the following files:
 - `-e` - Number of experiments. Default value 1.
 - `-r` - Random seed. If no random seed is provided, the experiments are not reproducible, since each time different random values are used. If a random seed is provided, the experiments are reproducible, and the same command line always produces the same result.
 - `-f` - Filter hypotheses. After a successful completion of stage 1, perform stage 2 only with the correct hypothesis. (In some cases, the first stage generates as many as 2,048 hypotheses.)
-- `-v` - Verbose. Permissible only if the number of experiments is 1 (which is the default). Outputs a summary of the rounds corresponding to bits 0-7 (or less if the bit size is less than 8).
+- `-v` - Verbose. Permissible only if the number of experiments is 1 (which is the default). Prints a detailed log of all the steps of the attack.
+
+Unless option `-f` is used, for every experiment a line is printed out. It includes the RNG seed used to generate the traces and the noise, the result (Success/Failure), and some additional information. If the number of experiments is large, using option `-f` is recommended in order to save time and suppress this printout.
+
+In any case, the two last lines of the printout of `test_sha2_attack.py` are:
+
+```bash
```

Comment:
`bash` keyword is used to highlight a shell commands sequence. In this case we have just some message printed to stdout, so it's better to use simple text highlighting.
```suggestion
```text
```